pep-0572.rst

```diff
@@ -275,16 +275,47 @@ i`` part establishes that ``i`` is local to the comprehension, but the
 ``i :=`` part insists that ``i`` is not local to the comprehension.
 The same reason makes these examples invalid too::
 
-    [[(j := j) for i in range(5)] for j in range(5)]
-    [i := 0 for i, j in stuff]
-    [i+1 for i in i := stuff]
+    [[(j := j) for i in range(5)] for j in range(5)] # compiler TargetScopeError
```

Comment:
What's the point of the `# compiler TargetScopeError` comment added to all these examples?

Reply:
The fact that these were all examples that *don't* compile seemed easy to miss when skimming the PEP, so I added a trailing comment on each one.

Reply:
I switched these over to the same `# INVALID` marker that the rest of the PEP already used.